modules/identity-provider-config-map.adoc

```diff
@@ -35,6 +35,24 @@ authority must be stored in the `ca.crt` key of the `ConfigMap` object.
 ----
 $ oc create configmap ca-config-map --from-file=ca.crt=/path/to/ca -n openshift-config
 ----
++
+[TIP]
+====
+You can alternatively apply the following YAML to create the config map:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ca-config-map
+  namespace: openshift-config
+type: Opaque
+data:
+  ca.crt: |
+    <CA_certificate_PEM>
```

Comment:
Compared with the oc command, after apply the yaml to create configmap, extra metadata.annotations is generated in the yaml output, is it expected?

```
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","data":{"ca.crt":"-----BEGIN CERTIFICATE-----END CERTIFICATE-----"},"kind":"ConfigMap","metadata":{"annotations":{},"name":"ca-config-map2","namespace":"openshift-config"},"type":"Opaque"}
```